pysatModels/models/sami2py_sami2.py

```diff
@@ -133,12 +133,18 @@ def load(fnames, tag=None, inst_id=None, **kwargs):
     """
 
     # Load data
-    data, meta = pysat.utils.load_netcdf4(fnames, pandas_format=False)
-
-    # Add time variable for pysat compatibilty
+    data, meta = pysat.utils.load_netcdf4(fnames, pandas_format=False,
+                                          epoch_name='ut')
+
+    # Add time variable for pysat compatibility
+    if 'time' in data:
+        label = 'time'
+    elif 'ut' in data:
+        label = 'ut'
```

Comment:
I updated the if/else to check against pysat version, since that is the underlying issue. I found a bug in pysat while working on this. The data_path in files can end up stale. I added the capability to update inst.files.data_path based upon what was loaded in stored file info. Needed it for multi-directory support. Anyway, that was incorrectly getting used to reassign stale file paths onto current Instruments. May cause issues here. It did for my local testing.